scripts/qa_crowbarsetup.sh

```diff
@@ -2007,8 +2007,13 @@ function reboot_nodes_via_ipmi()
                 if ipmitool -H $ip -U root -P $pw power status | grep -q "is off"; then
                     ipmitool -H $ip -U root -P $pw power on
                     sleep $((10 + RANDOM % 15))
+                else
+                    ipmitool -H $ip -U root -P $pw power off
+                    wait_for 5 2 "ipmitool -H $ip -U root -P $pw power status | grep -q 'is off'" "node to power on"
```

Comment:
well, the previous code was "power reset" (which is warm reboot and might not work). there shouldn't be a difference between power off/on and cycle. 

you did check for firmware updates already?